cpu/native/include/native_internal.h

```diff
@@ -96,6 +96,7 @@ extern int (*real_accept)(int socket, ...);
 extern int (*real_bind)(int socket, ...);
 extern int (*real_chdir)(const char *path);
 extern int (*real_close)(int);
+extern int (*real_fcntl)(int, ...);
```

Comment:
Shouldn't matter, since the second `int` is consumed by the `...` anyways (and this hack is used multiple times in this file), but fine, will do.